packages/store/package.json

```
@@ -14,7 +14,10 @@
     "lint-fix": "yarn lint:eslint --fix",
     "lint": "run-s --continue-on-error lint:*",
     "lint:types": "tsc --maxNodeModuleJsDepth 3 -p jsconfig.json",
-    "lint:eslint": "eslint ."
+    "lint:eslint": "eslint .",
+    "perf:build": "rollup --config test/rollup.config.js",
+    "perf:node": "NODE_OPTIONS='--jitless' node dist/bundle-perf-patterns.mjs",
+    "perf:xs": "../xsnap/moddable/build/bin/mac/release/xst -m dist/bundle-perf-patterns.mjs"
```

That is not portable. Also depends on `xst` having been built explicitly (it's not built as part of our regular build process). I think I'd rather have it reference an `xst` from the PATH and leave it to the reader on how to make that happen.

```suggestion
    "perf:xs": "xst -m dist/bundle-perf-patterns.mjs"
```

---

I'd like it to grab the XST for the version of moddable that we are using, rather than one installed in the enviroenment. That means it should use one built in the repo?

---

We'd need to start by updating the build scripts to build xst, and then find a way to make it work across both Linux and Mac. I do believe it should be a separate PR, hence why I suggested removing any path here.

I doubt anyone has xst in their PATH.